flocker/docs/version_code_block.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Sphinx extension to add a ``version-code-block`` directive
+
+This directive allows Flocker's release version to be inserted into code
+blocks.
+
+.. version-code-block:: console
+
+   $ brew install flocker-|RELEASE|
+"""
+
+from sphinx.directives.code import CodeBlock
+
+from flocker import __version__ as version
+from flocker.docs import parse_version
+
+
+class VersionCodeBlock(CodeBlock):
+    """
+    Similar to CodeBlock but replaces |RELEASE| with the latest release
+    version.
+    """
+    def run(self):
+        parsed_version = parse_version(version)
+        latest = parsed_version.release
+
+        if parsed_version.weekly_release is not None:
```

Comment:
I guess that this logic could be in a property of flocker.docs._version.FlockerVersion but it is kind of weird because it is trying to get the last version which e.g. Hombrew will have a script for. We don't make install scripts for documentation releases.